.expeditor/verify.pipeline.yml

```diff
@@ -8,6 +8,15 @@ steps:
 # Linting!
 #######################################################################
 
+  - label: "[unit] :linux: ARM Platform Support"
+    command:
+      - .expeditor/scripts/verify/run_native_cargo_test.sh
+    agents:
+      queue: docker-linux-arm64
+    plugins:
+      - docker#v3.13.0:
+          image: "rust:buster"
```

Comment:
Not sure how to get the rust version into the expeditor yaml. I could just directly put in the rust version. We should however remember to update it every time we upgrade the toolchain. It ultimately does not matter much because cargo will install the correct toolchain before running tests either way.